.github/workflows/ci.yaml

```diff
@@ -47,6 +43,7 @@ jobs:
         uses: dorny/paths-filter@v2
         id: filter
         with:
+          base: ${{ github.ref }}
```

Comment:
I don't think this works since it just compares the current commit with itself (which is a zero diff?). Maybe I'm wrong though